### ISSUE TITLE
Question about learning rate and batch size adjustment during the pre-training phase

### ISSUE THREAD

**Issue by Bo396543018** [facebookresearch/sapiens#177]
Hi，Thank you for your great work.
Could you tell me how the learning rate is adjusted during the pre-training phase for different model sizes and batch sizes?

**Comment by rawalkhirodkar** [comment]
@Bo396543018 batch sizes are set to max out GPU utilization.
Learning rate schedule is a linear warmup + cosine annealing with AdamW, [refer here](https://github.com/facebookresearch/sapiens/blob/08dce797f7b40f5b41388f518cac85535c3f5d13/pretrain/configs/sapiens_mae/humans_300m_test/mae_sapiens_1b-p16_8xb512-coslr-1600e_humans_300m_test.py#L95).
